regex-assembly/include/unix-shell-evasion-prefix.ra

```diff
@@ -16,8 +16,12 @@
   timeout
   watch
 ##!<
+##! =ifconfig
+=
 ##! ;ifconfig
 ;
+##! =ifconfig
+=
```

Comment:
Does the double-inclusion of `=` make sense, here? Or is there some regexp-assemble magic happening?

Reply:
No, that's a mistake. Thanks.